mne/io/tests/test_raw.py

```diff
@@ -257,3 +257,27 @@ def test_meas_date_orig_time():
     assert raw.annotations.orig_time is None
     assert raw.annotations.onset[0] == 0.5
     assert raw.annotations.duration[0] == 0.2
+
+
+def test_get_data_reject():
+    """Test if reject_by_annotation is working correctly."""
+    fs = 256
+    ch_names = ["C3", "Cz", "C4"]
+    info = create_info(ch_names, sfreq=fs)
+    raw = RawArray(np.zeros((len(ch_names), 10 * fs)), info)
+    raw.set_annotations(Annotations(onset=[2, 4], duration=[3, 2],
+                                    description="bad"))
+
+    with catch_logging() as log:
+        data = raw.get_data(reject_by_annotation="omit", verbose=True)
```

Comment:
how can the test pass on your machine? there is no verbose parameter. If fails for me too.

https://github.com/mne-tools/mne-python/blob/master/mne/io/base.py#L936

Reply:
Sorry, I meant that it passed one commit before.

Reply:
Cis run with level warning

Add the verbose parameter